server/cache_test.go

```diff
@@ -0,0 +1,181 @@
+package server
+
+import (
+	"net"
+
+	"github.com/golang/protobuf/proto"
+	. "github.com/pingcap/check"
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/kvproto/pkg/raftpb"
+)
+
+var _ = Suite(&testClusterCacheSuite{})
+
+type testClusterCacheSuite struct {
+	testClusterBaseSuite
+}
+
+func (s *testClusterCacheSuite) getRootPath() string {
+	return "test_cluster_cache"
+}
+
+func (s *testClusterCacheSuite) SetUpSuite(c *C) {
+	s.svr = newTestServer(c, s.getRootPath())
+	s.client = newEtcdClient(c)
+	deleteRoot(c, s.client, s.getRootPath())
+
+	go s.svr.Run()
+}
+
+func (s *testClusterCacheSuite) TearDownSuite(c *C) {
+	s.svr.Close()
+	s.client.Close()
+}
+
+func mustEqualStore(c *C, store1 *metapb.Store, store2 *metapb.Store) {
+	c.Assert(store1.GetId(), Equals, store2.GetId())
+	c.Assert(store1.GetAddress(), Equals, store2.GetAddress())
+}
```

Comment:
I mean you could use `c.Assert(store1, DeepEquals, store2)`, don't have to use these helper functions.